src/ship/PlayerShipController.cpp

```diff
@@ -505,7 +505,8 @@ void PlayerShipController::StaticUpdate(const float timeStep)
 		case CONTROL_MANUAL:
 			// maintain the thrust set by the station when undocking while the timeout lasts
 			// but only in manual mode
-			PollControls(timeStep, false, mouseMotion, act);
+			//Rotation damping forced for follow mode
+			PollControls(timeStep, m_followTarget && m_followMode == FOLLOW_ORI, mouseMotion, act);
```

Comment:
It turns out strange - you click on the rotation damping button, it is activated, but does not work. Maybe it's better to block it somehow?

Reply:
As strange as rotation damping in autopilot mode when damping is off. The autopilot mode is different from manual follow target roughly by automatically applied linear thrust.
I guess it is a subject for improvement of the lua flight gui to grey out the rotation button in automatic modes, but do we want such thing in this PR?